tornado_mysql/converters.py

```diff
@@ -123,7 +123,7 @@ def convert_datetime(obj):
         usecs = '0'
         if '.' in hms:
             hms, usecs = hms.split('.')
-        usecs = float('0.' + usecs) * 1e6
+        usecs = round(float('0.' + usecs) * 1e6)
```

Comment:
So I did some testing and determined that the MySQL documentation is a little misleading in some places.

In actuality, datetimes do have a fractional precision that you can specify for the number of decimal places to use. Thus, with testing, the `int(usec)` approach will not work because it's not always 6 characters as I previously inferred from other parts of the documentation.

Keeping it as ```round(float('0.' + usecs) * 1e6)``` should be ok!